Expand on x-xdn-t (#84)
moovweb-docs/xdn-docs: guides/response_headers.md

All times are in milliseconds.

|Name|Description|
|----|-----------|
|ot|The total time spent processing the request|
|or|The time spent matching routes at the edge pop|
|of|The time spent fetching the response at the edge pop|
|oc|The caching status for the edge pop|

> |ow|???|

Reviewer: ```suggestion
|ow|The free memory left to request processing at the edge pop|
|oq|The time spent in fetch queue during the request coalescing.
```

|st|The time spent processing the request at the shield pop|
|sr|The time spent matching routes at the shield pop|
|sf|The time spent waiting for a response from the serverless backend or origin, including request queueing|
|sc|The caching status for the shield pop|

> |sw|???|

Reviewer: ```suggestion
|sw|The free memory left to request processing at the shield pop|
|sq|The time spent at the shield pop in fetch queue during the request coalescing|
```

All times are in milliseconds.

|Name|Description|
|----|-----------|
|ot|The total time spent processing the request|
|or|The time spent matching routes at the edge pop|
|of|The time spent fetching the response at the edge pop|
|oc|The caching status for the edge pop|

> |ow|The free memory left after request processing at the edge pop|

Author: ```suggestion
|ow|The free memory in bytes on the edge pop after processing the request|
```

|oq|The time spent at the edge pop in fetch queue during the request coalescing|
|st|The time spent processing the request at the shield pop|
|sr|The time spent matching routes at the shield pop|
|sf|The time spent waiting for a response from the serverless backend or origin, including request queueing|
|sc|The caching status for the shield pop|

> |sw|The free memory left after request processing at the shield pop|

Author: ```suggestion
|ow|The free memory in bytes on the shield pop after processing the request|
```
